### ISSUE TITLE
Add payment by bank transfer or pay on delivery

### ISSUE THREAD

**Issue by MacLake** [mirumee/django-payments#134]
I’d like to have a payment method that does nothing but set the payment status to "pending" in order to process orders with payment by bank transfer or pay on delivery.

I’d rather have a new payment method than changing the shop’s logic in order to avoid problems with updating. We’d also like to accept Paypal in addition to bank transfer The choice would be easier to handle when you have another payment method.

**Comment by patrys** [comment]
I don't think this belongs here. From our experience cash-on-delivery and offline transfers require way more logic on the shop site than just setting the payment to pending. For such methods I'd even avoid creating a Payment object altogether until money is collected.

**Comment by devkral** [comment]
There must be some checks but such providers can be very handy.

**Comment by devkral** [comment]
see: https://github.com/mirumee/django-payments/pull/152 for an implementation.
